optimum/exporters/openvino/convert.py

```diff
@@ -615,9 +616,9 @@ def export_from_model(
             f"Compression of the weights to {ov_config.quantization_config} requires nncf, please install it with `pip install nncf`"
         )
 
-    library_name = _infer_library_from_model_or_model_class(model)
+    library_name = _infer_library_from_model_or_model_class(model, library_name=library_name)
     if library_name != "open_clip":
-        TasksManager.standardize_model_attributes(model)
+        TasksManager.standardize_model_attributes(model, library_name=library_name)
```

Comment:
why is this needed here ? (should only be an issue for open_clip no ?)

Reply:
I believe calling this explicitly is legacy code and can be removed now. standartize_model_attributes now is part of get_model_for_task
https://github.com/huggingface/optimum/blob/6335599da3bad65609620d1f64512e8241fd5aaa/optimum/exporters/tasks.py#L2303

Should I remove it?

Reply:
with passing library_name, such method should work with open_clip as well.
initial reason why I added this. library_name getting from model instance uses submodules and requires to have 'transformers`, while for janus some of model classes starts with own namespace and this check failed, however file based approach treats it as transformers based model.